cmd/root.go

```diff
@@ -240,6 +241,10 @@ func Detector(cmd *cobra.Command, cfg config.Config, source string) *detect.Dete
 		logging.Fatal().Err(err).Msg("")
 	}
 
+	if detector.MaxArchiveDepth, err = cmd.Flags().GetInt("max-archive-depth"); err != nil {
+		logging.Fatal().Err(err).Msg("")
```

Comment:
IMO the intent of [`.Send()`](https://pkg.go.dev/github.com/rs/zerolog#Event.Send) is clearer than `.Msg("")`, which could easily be a mistake or a forgotten "TODO".
```suggestion
		logging.Fatal().Err(err).Send()
```

Reply:
Agreed. I was just matching the style of the other spots in the file, but I'll refactor those as well.